elements/pfe-tabs/src/pfe-tab.scss

```
@@ -38,55 +42,50 @@ $LOCAL-VARIABLES: (
 
   // Inner style
   background-color: #{pfe-local(BackgroundColor)};
-  color: #{pfe-local(Color)};
+  color: pfe-local(Color);  // @todo replace with pfe(text-muted) in Kendall's color story
 
   cursor: pointer;
   text-align: #{pfe-local(TextAlign, center, $region: tab)};
-  
+
   // Exposing this for customization
   text-transform: #{pfe-local($cssvar: TextTransform, $region: tab, $fallback: none)};
   font-family:  #{pfe-var(font-family)};
   font-weight:  #{pfe-var(font-weight--normal)};
   font-size:    #{pfe-local(FontSize, $region: tab)};
-    
+
   .pfe-tab > * {
-    color: inherit !important;
     font-size: inherit;
     font-weight: inherit;
     margin: 0;
   }
 
   @at-root #{&}(:hover) {
-    --pfe-tabs--Color: #{pfe-var(text)};
+    --pfe-tabs--Color:  #{pfe-local(text-hover)};
     @include browser-query(ie11) {
-      color: #{pfe-var(text)};
+      color: #{pfe-local(text-hover)};
     }
   }
 
   // Active tab styles
   // TODO: ShadyCSS falls flat on this pseudo element in IE11 `:host(:focus-visible)`, add back when we drop ShadyCSS
   @at-root #{&}(:focus) {
-    --pfe-tabs--Color: #{pfe-var(text)};
-    outline: #{pfe-var(ui--focus-outline-width)} outset #{pfe-local(focus)};
-  }
-
-  @at-root #{&}([on="dark"]) {
-    --pfe-tabs--focus: #{pfe-var(link--focus--on-dark)};
+    color: #{pfe-var(text)};
   }
 
-  @at-root #{&}([on="saturated"]) {
-    --pfe-tabs--Color: #{darken($pfe-color--white, 20%)};
-    --pfe-tabs--focus: #{pfe-var(link--focus--on-saturated)};
-    --pfe-tabs--highlight: #{pfe-var(text--on-saturated)};
+  @at-root #{&}([on="dark"][aria-selected="false"][pfe-variant="wind"]:hover) {
+    --pfe-tabs__tab--BorderBottom: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} #{pfe-local(BorderColor--hover)};
+    @include browser-query(ie11) {
+      color: #{pfe-fetch(text)} !important;
+    }
   }
 
   @at-root #{&}([aria-selected="true"]) {
-    --pfe-tabs--Color: #{$pfe-color--black};
+    color: #{pfe-var(text)};
```

```suggestion
    color: pfe-var(text);
```

Same Q here, should this be `--pfe-tabs__tab--Color`?

---

I think this is no longer applicable, the file has changed too much, but let me know if this is still a concern